hedera-mirror-rest/tokens.js

```diff
@@ -34,6 +34,14 @@ import {CustomFeeViewModel, NftTransactionHistoryViewModel, NftViewModel} from '
 
 const {default: defaultLimit} = getResponseLimit();
 
+const customFeeSelect = `select jsonb_build_object(
+      'created_timestamp', ${CustomFee.CREATED_TIMESTAMP}::text,
+      'fixed_fees', ${CustomFee.FIXED_FEES},
+      'fractional_fees', ${CustomFee.FRACTIONAL_FEES},
+      'royalty_fees', ${CustomFee.ROYALTY_FEES},
+      'token_id', ${CustomFee.TOKEN_ID}::text,
```

Comment:
don't think the conversion is needed

```suggestion
      'token_id', ${CustomFee.TOKEN_ID},
```

Reply:
Thanks I have removed them.